elide-core/src/main/java/com/yahoo/elide/core/filter/Operator.java

```diff
@@ -316,6 +334,24 @@ public static Operator fromString(final String string) {
         return (T entity) -> false;
     }
 
+    private static <T> Predicate<T> isEmpty(String field, RequestScope requestScope) {
+        return (T entity) -> {
+            Object val = getFieldValue(entity, field, requestScope);
+            if (val == null) { return true; }
```

Comment:
null and empty do not represent the same state imo

Reply:
`For in memory stores, we'll need to check if the field is null OR if the collection is empty.` This is the requirement given in the issue.

Reply:
I agree - they are not the same thing.  But I think you could interpret this either way.

Reply:
if you look at it in a db table way, a column referring to children might be null or have a value with no associations .
consider 3 tables
TableA(a_id), TableB(b_id) and TableAB(ab_id, a_id, b_id).
You might end up with a row in TableAB such as ab_id = 1, a_id = 1 and b_id = null thus representing an empty collection.
You migth also end up having no row in TableAB thus representing a null collection.

I'm not sure if this is a concept worth representing in the =isempty= filter but that is how I can represent the distinction here.

Reply:
@jkusa Any thoughts on this one?

Reply:
We'll run a test and see what JPQL does.  The in-memory result should match the JPQL result.

Reply:
Makes sense. Is there a way where I could make the distinction between an absence of association between and an empty one then? I feel like is isNull and isEmpty operators would clash in some way.

Reply:
@PatRolland - Not quite following

Reply:
Doesn't matter I'm being picky 😄